app/access.py

```diff
@@ -24,7 +26,17 @@ def wrapper(self, *args, **kwargs):
         token_header = self.request.headers.get("Authorization", None)
         if token_header is not None and token_header.startswith("token "):
             token_id = token_header.replace("token", "").strip()
-            token = Token.query.get(token_id)
+            with DBSession() as session:
+                token = session.scalars(
+                    sa.select(Token)
+                    .options(
+                        joinedload(Token.created_by).options(
```

Comment:
Any reason not to have these loaded by default every time the token is retrieved?

Reply:
Maybe we should replace all the default eager loading with `selectin` loads? 
https://docs.sqlalchemy.org/en/14/orm/loading_relationships.html#select-in-loading

Reply:
I suppose it doesn't matter much, but I like the simplicity of that statement.

Reply:
and we cannot load Tokens by default because it doesn't pass the permissions checks since user1 cannot see the token of user2.